rfcs/0000-kuadrant-release-process.md

```diff
@@ -0,0 +1,205 @@
+|                                 | multicluster-gateway-controller-catalog images | [Quay.io](https://quay.io/repository/kuadrant/multicluster-gateway-controller-catalog) |
+| Policy Controller               | policy-controller images                       | [Quay.io](https://quay.io/repository/kuadrant/policy-controller)                       |
+| Kuadrant Operator               | kuadrant-operator images                       | [Quay.io](https://quay.io/repository/kuadrant/kuadrant-operator)                       |
+|                                 | kuadrant-operator-bundle images                | [Quay.io](https://quay.io/repository/kuadrant/kuadrant-operator-bundle)                |
+|                                 | kuadrant-operator-catalog images               | [Quay.io](https://quay.io/repository/kuadrant/kuadrant-operator-catalog)               |
+|                                 | kuadrant-operator manifests                    | [OperatorHub.io](https://operatorhub.io/operator/kuadrant-operator)                    |
+
+## Documentation
+
+It's important to  note that keeping the documentation up to date is a responsibility of the component maintainers, and
+it needs to be done before releasing a new version of the component. The importance of keeping a clear and up-to-date
+documentation is crucial for the success of the project.
+
+The documentation for the Kuadrant suite is compiled and available on the [Kuadrant website](https://kuadrant.io/). One
+can find the source of the documentation within each component repository, in the `docs` directory. However, making this
+information available on the website is a manual process, and should be done by the maintainers of the project. The
+process of updating the documentation is simple and consists of the following steps:
+
+1. Update the documentation in the corresponding component repository.
+2. In [https://github.com/Kuadrant/docs.kuadrant.io/](https://github.com/Kuadrant/docs.kuadrant.io/), update the value of
+the `import_url` within the `multirepo` in the `plugins` section of the [mkdocs.yml](https://github.com/Kuadrant/docs.kuadrant.io/blob/main/mkdocs.yml)
+file, to point to the tag or branch of the component repository that contains the updated documentation.
+3. Once the changes are merged to main, the workflow that updates the website will be triggered, and the documentation
+will be updated.
+4. If for some reason it's needed to trigger the workflow manually, one can do it from the GitHub Actions tab in the
+[docs.kuadrant.io](https://github.com/Kuadrant/docs.kuadrant.io/actions/workflows/build.yaml) (`Actions > ci > Run Workflow`).
+
+## Communication
+
+Another important aspect of releasing a new version of the Kuadrant suite is the communication with the community and
+other teams within the organization. A few examples of the communication channels that need to be updated are:
+
+- Changelog generation
+- Release notes
```

Comment:
I like how Istio is doing this.

Release announcements

![Screenshot 2024-02-16 at 14-59-30 Release Announcements](https://github.com/Kuadrant/architecture/assets/881529/5a9c3800-e56d-487d-9b5d-a90433f3bb52)


X.Y.x patch releases


![Screenshot 2024-02-16 at 15-00-23 1 20 x Releases](https://github.com/Kuadrant/architecture/assets/881529/29e3dff4-d603-4703-8a3f-455e4a94464f)

New release notes

![Screenshot 2024-02-16 at 15-00-52 Announcing Istio 1 20 3](https://github.com/Kuadrant/architecture/assets/881529/bd7b20d5-c973-4740-884f-9caec4ffe72f)